src/Conversation.php

```diff
@@ -0,0 +1,242 @@
+<?php
+/**
+ * This file is part of the TelegramBot package.
+ *
+ * (c) Avtandil Kikabidze aka LONGMAN <akalongman@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Longman\TelegramBot;
+
+/**
+ * Class Conversation
+ *
+ * Only one conversation can be active at any one time.
+ * A conversation is directly linked to a user, chat and the command that is managing the conversation.
+ */
+class Conversation
+{
+    /**
+     * All information fetched from the database
+     *
+     * @var array
+     */
+    protected $conversation = null;
+
+    /**
+     * Notes stored inside the conversation
+     *
+     * @var array
+     */
+    protected $protected_notes = null;
+
+    /**
+     * Notes to be stored
+     *
+     * @var array
+     */
+    public $notes = null;
```

Comment:
A suggestion:
Instead of a public array, I think having a `getNote` and `setNote` method makes it more robust and less prone for error. This way we can take care of the array management so the user doesn't have to.

Could you please explain why you chose to use a public var? I'd like to learn your view :smiley: